components/bl-audio-player-component/component.json

```diff
@@ -0,0 +1,86 @@
+{
+  "id": "c_8f1b55ecfb0b86c23eb39de3d72356a9",
+  "name": "Audio Player",
+  "description": "Audio Player is a component for playing a variety of audio URLs.",
+  "showInToolbox": true,
+  "faIcon": "music",
+  "mainJS": "dist/index.js",
+  "type": "custom",
+  "category": "Custom Components",
+  "properties": [
+    {
+      "name": "audioUrl",
+      "label": "Audio Url",
+      "type": "text",
+      "handlerId": "audioUrlLogic",
+      "handlerLabel": "Audio Url Logic",
+      "dataBinding": true,
+      "handlerDescription": "This is a handler for the logic to determine the url of the audio. Value returned by the handler will be used as the audio url."
+    },
+    {
+      "name": "audioTitle",
+      "label": "Audio Title",
+      "type": "text",
+      "handlerId": "audioTitleLogic",
+      "handlerLabel": "Audio Title Logic",
+      "dataBinding": true,
+      "handlerDescription": "This is a handler for the logic to determine the title of the audio. Value returned by the handler will be used as the audio title."
+    },
+    {
+      "type": "checkbox",
+      "name": "multipleAudioPlayer",
+      "label": "Multiple Audio Player",
+      "defaultValue": false
+    },
+    {
+      "type": "checkbox",
+      "name": "playerVisibility",
+      "label": "Player Visibility",
+      "defaultValue": true
+    },
+    {
+      "type": "checkbox",
+      "name": "trackNavigationVisibility",
+      "label": "Track Navigation Visibility",
+      "defaultValue": true
+    },
+    {
+      "type": "checkbox",
+      "name": "titleVisibility",
+      "label": "Title Visibility",
+      "defaultValue": true
+    },
+    {
+      "type": "number",
+      "name": "defaultVolume",
+      "label": "Default Volume",
+      "defaultValue": "50"
+    },
+    {
+      "type": "checkbox",
+      "name": "autoPlay",
+      "label": "Auto Play"
+    },
+    {
+      "type": "checkbox",
+      "name": "repeatOne",
+      "label": "Repeat One"
+    },
+    {
+      "handlerId": "audioDataLogic",
```

Comment:
What is we rework singe/multiple type of player.

What should be done:

- remove this handler and multipleAudioPlayer property.
- if from audioUrlsLogic comes string - that means it is single player, if array - multiple.

Contact me if something not clear.